lib/widgets/compose_box.dart

```diff
@@ -930,30 +930,22 @@ class _SendButtonState extends State<_SendButton> {
 
   @override
   Widget build(BuildContext context) {
-    final disabled = _hasValidationErrors;
-    final colorScheme = Theme.of(context).colorScheme;
+    final designVariables = DesignVariables.of(context);
```

Comment:
```
compose_box: Use new send button color, shape and icon
```

nit: this commit doesn't change the send button's icon; it was changed (to `ZulipIcons.send`) in an earlier commit.